src/main/java/teammates/ui/webapi/GetFeedbackSessionsAction.java

```diff
@@ -69,15 +70,23 @@ public JsonResult execute() {
         String entityType = getNonNullRequestParamValue(Const.ParamsNames.ENTITY_TYPE);
 
         List<FeedbackSessionAttributes> feedbackSessionAttributes;
+        List<StudentAttributes> students = new ArrayList<>();
         List<InstructorAttributes> instructors = new ArrayList<>();
 
         if (courseId == null) {
             if (entityType.equals(Const.EntityType.STUDENT)) {
-                List<StudentAttributes> students = logic.getStudentsForGoogleId(userInfo.getId());
+                students = logic.getStudentsForGoogleId(userInfo.getId());
                 feedbackSessionAttributes = new ArrayList<>();
                 for (StudentAttributes student : students) {
                     feedbackSessionAttributes.addAll(logic.getFeedbackSessionsForCourse(student.getCourse()));
                 }
+                if (students.isEmpty()) {
+                    throw new EntityNotFoundException(NO_SUCH_USER_FOUND);
+                }
+                String emailAddress = students.get(0).getEmail();
```

Comment:
Cannot assume all students in the list have the same email address. Move this part of the logic into the for loop lines 80-82.